app/sliders/config_custom.yaml

```diff
@@ -73,10 +73,11 @@ pie_custom:
   plot_type: pie
   general:
     height: 350
+    width: 380
     title: Pie Chart
     toolbar_location: null
     tools: hover
-    tooltips: '@x: @y'
```

Comment:
apparently we have different linting/formatting configs. we should get a proper standard here

Reply:
yes, i want to refactor the config eventually, this was very quick and dirty

Reply:
i normally always use double quote, so this was not intended, thanks for the check!